### ISSUE TITLE
spring boot2.2.0： jasypt-spring-boot2.1.0 Failed to bind properties under

### ISSUE THREAD

**Issue by lunxian8** [ulisesbocchio/jasypt-spring-boot#161]
Upgrading spring boot2.2.0 found that a reference to jasypt2.1 caused the spring-@configurationproperties injection configuration error

**Comment by huxudong** [comment]
application.yml 
add  proxy-property-sources: true 

example:

```
jasypt:
  encryptor:
    password: password
    proxy-property-sources: true
```

**Comment by zuoqi1993125** [comment]
> application.yml
> add proxy-property-sources: true
> 
> ### example:
> ```yaml
> jasypt:
>   encryptor:
>     password: password
>     proxy-property-sources: true
> ```

我加了这个配置,但是没有生效,依然绑定properties失败

**Comment by ulisesbocchio** [comment]
thanks @huxudong @zuoqi1993125 
@lunxian8 was `proxy-property-sources` helpful?

**Comment by lunxian8** [comment]
> thanks @huxudong @zuoqi1993125
> @lunxian8 was `proxy-property-sources` helpful?

added the `proxy - property - sources: true`
But there are still errors

**Comment by ulisesbocchio** [comment]
fixed in 3.0.0
